bokehjs/src/compiler/compile.ts

```diff
@@ -240,7 +203,7 @@ const compile_and_resolve_deps = (input: {code: string, lang: string, file: stri
 if (argv.file != null) {
   const input = {
     code: fs.readFileSync(argv.file as string, "utf-8"),
-    lang: (argv.lang as string | undefined) || "coffeescript",
+    lang: argv.lang as string,
```

Comment:
The original needs to be restored with `coffeescript` replaced with `typescript`.